appinventor/components/src/com/google/appinventor/components/runtime/Web.java

```diff
@@ -364,6 +395,9 @@ public void run() {
         } catch (FileUtil.FileException e) {
           form.dispatchErrorOccurredEvent(Web.this, METHOD,
               e.getErrorMessageNumber());
+        } catch (RequestTimeoutException e) {
+          form.dispatchErrorOccurredEvent(Web.this, METHOD,
+                  ErrorMessages.ERROR_WEB_REQUEST_TIMED_OUT, webProps.urlString);
```

Comment:
Indentation here should be +4 from `f` in `form` (under `.`).  Also make this update for the other places in this file.

Reply:
Corrected all of these